tsl/test/max_workers_8.conf.in

```diff
@@ -0,0 +1,30 @@
+# This section has to be equivalent to test/postgresql.conf
+shared_preload_libraries=timescaledb
+max_worker_processes=16
```

Comment:
The name `max_workers_8.conf.in ` seems to be a bit misleading. Should it be workers_16?

Reply:
The "8 workers" refer to the background workers for TimescaleDB that is later in the file. This is the total number of workers, including non-TimescaleDB workers. Probably good to change the name to make that clear. Does `max_bgw_8` sounds fine?

Reply:
Oh, I see. Yes, max_bgw_8 sounds better.

Reply:
Done